### DISCUSSION TITLE
Convert Flax model to PyTorch

### DISCUSSION THREAD

**Discussion by HosseinMA96** [google/flax#2832] [Q&A]
Hi,

I have three image classifiers trained in Flax, with their state/params as a frozendict. Two of the model are custom CNNS with common layers, consisted of 1m parameters and another one is a ResNet18 with 11m parameters. I was wondering how could I convert this model to Pytorch and obtain an identical model?

Thanks.

**Comment by cgarciae** [comment]
I haven't used it but I know HuggingFace has these utilities which may help:

https://github.com/huggingface/transformers/blob/main/src/transformers/modeling_flax_pytorch_utils.py
